packages/fuel-indexer-tests/components/indices/fuel-indexer-test/Cargo.toml

```diff
@@ -11,8 +11,9 @@ crate-type = ['cdylib']
 fuel-indexer-macros = { version = "0.3", path = "../../../../fuel-indexer-macros", default-features = false }
 fuel-indexer-plugin = { version = "0.3", path = "../../../../fuel-indexer-plugin" }
 fuel-indexer-schema = { version = "0.3", path = "../../../../fuel-indexer-schema", default-features = false }
-fuel-tx = "0.23"
-fuels-core = "0.30"
-fuels-types = "0.30"
+fuel-tx = "0.26.0"
+fuels-core = { git = "https://github.com/FuelLabs/fuels-rs", branch = "segfault_magnet/wasm_friendly_abigen", default-features = false }
+fuels-macros = {   git = "https://github.com/FuelLabs/fuels-rs", branch = "segfault_magnet/wasm_friendly_abigen"}
+fuels-types = {   git = "https://github.com/FuelLabs/fuels-rs", branch = "segfault_magnet/wasm_friendly_abigen" , default-features = false }
```

Comment:
@deekerno  I don't think the formatting here matches the formatting of any of our other TOML files (spacing)